### DISCUSSION TITLE
Manager IP on Sensor

### DISCUSSION THREAD

**Discussion by DmitriyusGit** [Security-Onion-Solutions/securityonion#4100] [Unsupported Versions] [answered]
Where is the Manager IP stored on the Sensor? 

During setup it asks for the Manager IP, but since the install the IP needed to be changed and it needs to be updated on the Sensor now. Thank you!

**Comment by b4b857f6ee** [comment]
If i remember well they didn't really support changing the ip for now :/

**Reply by b4b857f6ee** [reply]
https://github.com/Security-Onion-Solutions/securityonion/discussions/3156

**Comment by TOoSmOotH** [comment, marked as answer]
The fastest way would to be re-run setup on the remote sensor.
